content/en/security_platform/application_security/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Application Security
+kind: documentation
+description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
+is_beta: true
+disable_sidebar: true
+further_reading:
+- link: "/security_platform/application_security/setup_and_configure/#compatibility"
+  tag: "Documentation"
+  text: "Learn more about language and framework compatibility"
+---
+
+<div class="alert alert-warning">
+Application Security is in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
+</div>
+
+{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="A security signal panel in Datadog, which displays attack flows and flame graphs" width="75%">}}
+
+Datadog Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).
+
+Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever an attack targets your production system, or a vulnerability is triggered from the code.
+
+For example, when a `HIGH` or `CRITICAL` threat is detected, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or PagerDuty to notify your team and provide real-time context around threats.
```

Comment:
On 2nd read, I understand that only HIGH/CRITICAL severity threats generate signals. I think it'd read better as:

> When a threat is detected, a security signal is generated in Datadog. For `HIGH` or `CRITICAL` severity security signals, notifications can be sent to Slack, email, or PagerDuty to notify your team and provide real-time context around threats.

Wdyt?

Reply:
```suggestion
When a threat is detected, a security signal is generated in Datadog. For `HIGH` or `CRITICAL` severity security signals, notifications can be sent to Slack, email, or PagerDuty to notify your team and provide real-time context around threats.
```